packages/x-charts/src/Gauge/Gauge.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import composeClasses from '@mui/utils/composeClasses';
+import { GaugeContainer, GaugeContainerProps } from './GaugeContainer';
+import { GaugeValueArc } from './GaugeValueArc';
+import { GaugeReferenceArc } from './GaugeReferenceArc';
+import { GaugeClasses, getGaugeUtilityClass } from './gaugeClasses';
+import { GaugeValueText, GaugeValueTextProps } from './GaugeValueText';
+
+export interface GaugeProps extends GaugeContainerProps, Pick<GaugeValueTextProps, 'text'> {
+  classes?: Partial<GaugeClasses>;
+  children?: React.ReactNode;
+}
+
+const useUtilityClasses = (props: GaugeProps) => {
+  const { classes } = props;
+
+  const slots = {
+    root: ['root'],
+    valueArc: ['valueArc'],
+    referenceArc: ['referenceArc'],
+    valueText: ['valueText'],
+  };
+
+  return composeClasses(slots, getGaugeUtilityClass, classes);
+};
+
+function Gauge(props: GaugeProps) {
+  const { text, children, ...other } = props;
```

Comment:
We need to exclude `classes` here.
Currently, when passing `classes` it is also applied directly to the root element. 🙈